src/meta/proto-conv/tests/it/proto_conv.rs

```diff
@@ -491,5 +509,19 @@ fn test_load_old() -> anyhow::Result<()> {
         assert_eq!(want, got);
     }
 
+    // TableCopiedFileInfo is loadable
+    {
+        let stage_file: Vec<u8> = vec![
+            10, 4, 101, 116, 97, 103, 16, 128, 8, 26, 23, 50, 48, 49, 52, 45, 49, 49, 45, 50, 57,
+            32, 49, 50, 58, 48, 48, 58, 48, 57, 32, 85, 84, 67, 160, 6, 6, 168, 6, 1,
+        ];
+        let p: pb::TableCopiedFileInfo =
```

Comment:
Add a version suffix to let the future you know what the time it is to remove a test case when a feature is no longer supported.